example/libclang-example/generated_bindings.dart

```diff
@@ -7665,9 +7665,12 @@ typedef NativeClang_getCString = ffi.Pointer<ffi.Char> Function(
     CXString string);
 typedef DartClang_getCString = ffi.Pointer<ffi.Char> Function(CXString string);
 typedef NativeClang_disposeString = ffi.Void Function(CXString string);
+typedef NativeClang_disposeString_dart = void Function(CXString string);
 typedef DartClang_disposeString = void Function(CXString string);
```

Comment:
Hmmm, yeah. I guess this PR is sort of duplicating what the `exposeFunctionTypedefs` flag does. The difference is that `exposeFunctionTypedefs` is only for functions, is conditionally enabled by the config, and these typedefs don't correspond to any typedefs in the header (and they use a slightly different naming convention, putting `Dart` and `Native` at the start). We'll need to unify them somehow.

It's also a bit weird because `exposeFunctionTypedefs` defines 2 separate `Typealias`es for the function, whereas in this PR I'm having a single `Typealias` generate 2 typedefs.

Reply:
Perhaps we can skip generating the `<func>_dart` typedef for the typealiases added via `exposeFunctionTypedefs`? This would avoid the naming change for the currently generated `Dart<func>` typedef

Reply:
Yep. I'll refactor the `exposeFunctionTypedefs` to codegen 2 `typedef` expressions from a single `Typealias`, and change my newly added aliases to prefix `Dart` rather than sufixing `_dart`. I'll also have to update https://github.com/dart-lang/ffigen/pull/621 to be consistent, changing the `_function` suffix to `Function`.

Reply:
Done. PTAL